content/en/monitors/configuration/_index.md

```diff
@@ -181,8 +181,8 @@ The selected behavior is applied when a monitor's query does not return any data
 
 The `Evaluate as zero` and `Show last known status` options are displayed based on the query type:
 
-- **Evaluate as zero:** This option is available for monitors using `Count` queries without the `default_zero()` function.
-- **Show last known status:** This option is available for monitors using any other query type than `Count`, for example `Gauge`, `Rate`, and `Distribution`, as well as for `Count` queries with `default_zero()`.
+- **Evaluate as zero:** This option is available for monitors using `Count` queries without the `default_zero()` function. This option is also available for Metrics monitors that exclusively use queries with either `Sum` space aggregations or the `default_zero()` function. 
```

Comment:
Suggestion to clarify that the first part doesn't apply to metrics monitors, and also mention some other cases where "evaluate as zero" can be used
```suggestion
- **Evaluate as zero:** This option is available for monitors using `Count` queries without the `default_zero()` function. For Metrics monitors, this option is available for queries where each metric either uses the `sum` space aggregation, is a count of distribution values, or uses one of those functions: `default_zero()`, `count_nonzero()` or `count_not_null()`. 
```